plugins/lookup/etcd3.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# (c) 2020, SCC France, Eric Belhomme <ebelhomme@fr.scc.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    author:
+    - Eric Belhomme <ebelhomme@fr.scc.com>
+
+    lookup: etcd3
+
+    version_added: "2.9.6"
+
+    short_description: get info from an etcd3 server
+
+    description:
+    - Retrieves data from an etcd3 server
+
+    options:
+        _terms:
+            description:
+            - the list of keys to lookup on the etcd3 server
+            type: list
+            elements: string
+            required: True
+        prefix:
+            description:
+            - look for a key or for a prefix key
+            type: boolean
```

Comment:
```suggestion
            type: bool
```

Reply:
@Akasurde I first wanted to mention that too, but then I saw that other lookup plugins also use `boolean` and not `bool` (they also use `string` and `integer` instead of `str` and `int`). Do you know whether both values are possible (in each case), or whether only one is valid?

Reply:
good question, actually I made the same assumptions than @felixfontein when I wrote this piece of code. A clarification would be welcome ;)

Reply:
According to [the code](https://github.com/ansible/ansible/blob/devel/lib/ansible/config/manager.py#L54-L160), both are possible. @Akasurde is there a reason why you are suggesting the short form? Is there an initiative to unify type names (see first comment in code)?

Reply:
fixed in commit 29c84960aba5dd6f88bb39fc10f1cbe8be0e31a3